services/order.py

```diff
@@ -0,0 +1,34 @@
+from db.models import Order, Ticket
+from django.db import transaction
+from django.contrib.auth import get_user_model
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None) -> None:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user=user)
```

Comment:
The Order model is being instantiated correctly. The attributes are defined before the Meta class. Good job!